src/overlays/actors/ovl_Bg_Ddan_Jd/z_bg_ddan_jd.c

```diff
@@ -15,11 +15,13 @@ void BgDdanJd_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgDdanJd_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgDdanJd_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80870B88(BgDdanJd* this, GlobalContext* globalCtx);
-void func_80870D2C(BgDdanJd* this, GlobalContext* globalCtx);
-void func_80870F00(BgDdanJd* this, GlobalContext* globalCtx);
+void BgDdanJd_Idle(BgDdanJd* this, GlobalContext* globalCtx);
+void BgDdanJd_MoveEffects(BgDdanJd* this, GlobalContext* globalCtx);
+void BgDdanJd_Move(BgDdanJd* this, GlobalContext* globalCtx);
```

Comment:
```suggestion
void BgDdanJd_Idle(BgDdanJd* this, GlobalContext* globalCtx);
void BgDdanJd_Move(BgDdanJd* this, GlobalContext* globalCtx);
```
unneeded, defined before used